app/Services/Manager.php

```diff
@@ -608,4 +608,24 @@ public function sendEmail($user, $contest, $params)
             return null;
         }
     }
+
+    /**
+     * Finds if user exists in a competition for a given contest
+     *
+     * @param app\Models\Contest         $contest
+     * @param string                     $id
+     * @return array|null
+     */
+    public function findUserInContest($contest, $id)
+    {
+        $userCompetitions = $contest->competitions->filter(function ($competition, $key) use ($id) {
+            $users = $competition->users->where('northstar_id', $id);
```

Comment:
Maybe rename this from `$users` to `$result`? It's a bit confusing because it seems like it'll return more than one user, but this is find a specific single user in an series of potential competitions they might be in, it just happens to return a collection or whatever.